daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/Node.scala

```diff
@@ -64,6 +67,9 @@ object Node {
       copy(coid = f(coid))
 
     override def mapNodeId[Nid2](f: Nothing => Nid2): NodeFetch[Cid] = this
+
+    override def requiredAuthorizers: Set[Party] = actingParties.getOrElse(Set.empty)
```

Comment:
This is also dangerous -- is it actually correct?

Reply:
It's not correct for versions of LF transactions that have authorized fetches, but no no fetch actors. But I'm unsure what to do; let's discuss when you have time.